src/psyclone/tests/gocean1p0_test.py

```diff
@@ -972,6 +974,20 @@ def test00p1_kernel_wrong_meta_arg_count():
               api="gocean1.0")
 
 
+def test00p1_invoke_kernel_using_const_scalar():  # pylint:disable=invalid-name
+    '''Check' that using a const scalar as parameter works.'''
+    filename = os.path.join(os.path.dirname(os.path.abspath(__file__)),
+                            "test_files", "gocean1p0",
+                            "test00.1_invoke_kernel_using_const_scalar.f90")
+    _, invoke_info = parse(filename, api="gocean1.0")
+    out = str(PSyFactory(API).create(invoke_info).gen)
+    # Old versions of PSyclone tried to declare '0' as a variable:
+    # REAL(KIND=wp), intent(inout) :: 0
+    # Make sure this is not happening anymor
+    import re
+    assert re.search(r"\s*real.*:: *0", out, re.I) is None
+
+
```

Comment:
Please add a test for passing a real quantity by value. There may already be some test Fortran code where this is done.

Reply:
OK, I am sure I misunderstood something ... I've added a test with a kernel using a by-value parameters, but I am somehow that doesn't make any sense tbh, as far as I can see this has nothing to do with this pull request. I've added this as a separate commit, so it's easy to undo. Just let me know what I missed here ;)